packages/core/src/shaderlib/extra/particle.vs.glsl

```diff
@@ -18,6 +18,7 @@ attribute float a_StartSpeed;
 
 //#if defined(COLOR_OVER_LIFETIME) || defined(RENDERER_COL_RANDOM_GRADIENTS) || defined(RENDERER_SOL_RANDOM_CURVES) || defined(RENDERER_SOL_RANDOM_CURVES_SEPARATE) || defined(ROTATION_OVER_LIFE_TIME_RANDOM_CONSTANTS) || defined(ROTATION_OVER_LIFETIME_RANDOM_CURVES)
     attribute vec4 a_Random0;
+    attribute vec3 a_Random2;
```

Comment:
Completely supplement the xyzw of a_Random0, a_Random1, and a_Random2 in Shader. Who will use it?